keanu-project/src/main/java/io/improbable/keanu/vertices/dbl/probabilistic/ExponentialVertex.java

```diff
@@ -21,45 +21,45 @@
 
 public class ExponentialVertex extends DoubleVertex implements ProbabilisticDouble, SamplableWithManyScalars<DoubleTensor> {
 
-    private final DoubleVertex lambda;
+    private final DoubleVertex rate;
```

Comment:
why are you changing the name of this field?

Reply:
`lambda` is a keyword in Python so the easy fix was to change the parameter name in the constructor. I just changed the field name too for consistency

Reply:
Given the domain Keanu operates in, are we likely to come across a few instances of "lambda"s as we move forward?  Do we have an obvious error message if the lambda thing pops up?  Perhaps we want to add one to the code generator if not?

Reply:
Yeah, that might be a good idea... The only reason I caught it was because one of the python tests used Exponential. Otherwise it could have gone unnoticed!